tfjs-backend-webgpu/src/backend_webgpu_test.ts

```diff
@@ -567,3 +567,120 @@ describeWebGPU('create tensor from GPUBuffer', () => {
 describeWebGPU('create tensor from GPUBuffer with zero copy', () => {
   createTensorFromGPUTest(true);
 });
+
+describeWebGPU('getShader ', () => {
+  let flag: boolean;
+  const conv2dSpec = {
+    'name': 'conv2d',
+    'inputs': [
+      {'shape': [4, 4, 1], 'dtype': 'float32'},
+      {'shape': [1, 1, 1, 3], 'dtype': 'float32'}
+    ],
+    'args': [[2, 2], 'same', 'NHWC', [1, 1]]
+  };
+
+  const addSpec = {
+    'name': 'add',
+    'inputs':
+        [{'shape': [6], 'dtype': 'float32'}, {'shape': [6], 'dtype': 'float32'}]
+  };
+
+  const addSpec2 = {'name': 'add', 'inputs': [{'shape': [6]}, {'shape': [6]}]};
+  const sinSpec = {
+    'name': 'sin',
+    'inputs': [{'shape': [6], 'dtype': 'float32'}]
+  };
+  const badSpec = {
+    'name': 'opNotExist',
+    'inputs': [{'shape': [6]}, {'shape': [6]}]
+  };
+
+  beforeAll(() => {
+    flag = tf.env().getBool('WEBGPU_CPU_FORWARD');
+    tf.env().set('WEBGPU_CPU_FORWARD', true);
```

Comment:
Since you have set  `tf.env().set('WEBGPU_CPU_FORWARD', true);` in `getShader`, `beforeAll` and `afterAll` can be removed?